src/vit-servicing-station/vit-servicing-station-tests/src/common/snapshot.rs

```diff
@@ -124,6 +125,20 @@ pub struct VoterInfo {
     pub voter_info: Vec<VotingPower>,
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+pub struct DelegatorInfo {
```

Comment:
Why not use already defined `DelegatorInfo` struct from https://github.com/input-output-hk/catalyst-core/blob/8071368992ae04aa56f9ce15ccb144c0fdee2e6b/src/vit-servicing-station/vit-servicing-station-lib/src/v0/endpoints/snapshot/mod.rs#L106 ?

Reply:
right:)